### DISCUSSION TITLE
Why px for spacing and rems for others

### DISCUSSION THREAD

**Discussion by kvpasupuleti** [tailwindlabs/tailwindcss#8841] [General]
I have gone through various design systems for deriving the units for our design system.

Why is tailwind using `px` for paddings and margins and `rems` for others including other spacings and sizing?

Any base for this decision here?

**Comment by adamwathan** [comment]
Hey! We use rem for padding and margin, not pixels.

**Reply by kvpasupuleti** [reply]
@adamwathan 
But, according to these docs, padding and margin are mentioned in `px`

https://tailwindcss.com/docs/padding
https://tailwindcss.com/docs/margin

**Reply by cannap** [reply]
yes but only the 0 values

**Comment by yevgeniy-belov** [comment]
Im wondering why using rems for spacing - doesn't it make the custom font size feature of the browsers less useful?
If a user wants to make _everything_  bigger she can Zoom In but if she wants to make _only the texts_ bigger, without consuming even more real estate, she should be able to do so. WDYT?
